cloudformation/template.yaml

```diff
@@ -46,12 +46,12 @@ Parameters:
   DevPortalCustomersTableName:
     Type: String
     Description: The name of the DynamoDB Customers table.
-    Default: "DevPortalCustomers"
+    Default: 'DevPortalCustomers'
```

Comment:
do we have a way to auto-fix some of these following our linting rules?

Reply:
The linter I've been using is functionality only, not style. I'm sure someone's written a CFN style linter, might be worth adding after we have the functional one built-in?